pkg/registers/marshalling.go

```diff
@@ -150,13 +154,14 @@ func getRegister8Parser(regID RegisterID) func(ui8 uint8) Register {
 	return nil
 }
 
-// Unmarshal constructs register from it's id and marshalled value
-func Unmarshal(id RegisterID, b []byte) (Register, error) {
+// ValueFromBytes constructs register from it's id and marshalled value
+// TODO: use dynamic conversion instead of hard-code
+func ValueFromBytes(id RegisterID, b []byte) (Register, error) {
```

Comment:
nit RegisterFromBytes.

The counter call was named Marshal"Value" as it marshalled only the value of the register